pipeline/bedstat.py

```diff
@@ -63,6 +69,41 @@
 if not args.just_db_commit:
     pm = pypiper.PipelineManager(name="bedstat-pipeline", outfolder=outfolder,
                                  args=args)
+
+    # Produce bigBed (bigNarrowPeak) file from peak file 
+    bigNarrowPeak = os.path.join(outfolder, fileid + ".bigBed")
+    temp = tempfile.NamedTemporaryFile(dir=outfolder, delete=False)
+    print ("test bigbed saving path: ", bigNarrowPeak)
+    print ("test chrom.sizes path: ", args.chrom_size)
+    if not os.path.exists(bigNarrowPeak):
+        df = pd.read_csv(args.bedfile, sep='\t', usecols=[0,1,2], header=None,
+                            names=("V1","V2","V3","V4","V5","V6",
+                                    "V7","V8","V9","V10")).sort_values(by=["V1","V2"])
+        df.to_csv(temp.name, sep='\t', header=False, index=False)
```

Comment:
It feels like we can avoid using pandas here. It's a dependency-heavy library that will become cumbersome if we decide to containerize the pipeline in the future. See this: https://github.com/databio/bedhost/issues/35